lib/controllers/chores.js

```diff
@@ -0,0 +1,37 @@
+const { Router } = require('express');
+const Chore = require('../models/Chore');
+
+module.exports = Router()
+  .post('/', async (req, res) => {
+    const chore = await Chore.insert(req.body);
+    res.send(chore);
+  })
+
+  .get('/', async (req, res) => {
+    const chores = await Chore.findAll();
```

Comment:
We will cover error handling more in depth soon, but we should atleast have a try catch on all of our routes passing the error into next to ensure that the application won't crash if an error occurs